### ISSUE TITLE
Documentation update for Chrome/NVDA

### ISSUE THREAD

**Issue by NSoiffer** [mathjax/MathJax#1867]
On http://docs.mathjax.org/en/latest/misc/accessibility-features.html?highlight=accessibility, the following line is out of date:

NVDA | Chrome | any | DNA | Chrome issues prevent MathML support by NVDA

As of sometime in August 2017, Chrome and NVDA now work well together reading MathML in a web page. The line should be updated.

**Comment by pkra** [comment]
Filed at https://github.com/mathjax/MathJax-docs/issues/211.

Closing here.
